src/tables.jl

```diff
@@ -29,6 +85,55 @@ function Base.copy(c::Column{T, T}) where {T <: Union{String, Union{String, Miss
     return A
 end
 
+function Base.copy(c::Column2{T, S}) where {T <: Union{String, Union{String, Missing}}, S <: Union{PooledString, Union{PooledString, Missing}}}
```

Comment:
Same here: it looks like you could at least factor out a large part of code which is common with `Column`.